### DISCUSSION TITLE
In-Class Group Assignment

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#13] [Assignments]
## Submission
Post a link to your [in-class group assignment](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_04/Materials/inclass_group_exercise.md) as a comment below:

**Comment by edmondywen** [comment]
https://www.figma.com/file/mPfds02PSB0tfbW4RdX0a5/Little_Tokyo-Survey_Map-v1?node-id=0%3A1

**Comment by abi-200** [comment]
https://drive.google.com/file/d/19d2GijFKw2I79uxX5X8wkxC_DVTPSPu7/view?usp=sharing

**Comment by emilysuh** [comment]
https://drive.google.com/file/d/1HGcM2-ow97CBOftJaMDBpHOp_rsLdivi/view?usp=sharing

**Comment by perezsara59** [comment]
https://miro.com/app/board/o9J_l6S1kYY=/

**Comment by eandrusenki** [comment]
https://miro.com/app/board/o9J_l5sjgqo=/
